src/content/blog/2025-04-29-introducing-open-payments-php.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: "Introducing Open Payments PHP: A New Bridge for Financial Interoperability"
+description: "The first work week of the season"
+date: 2025-04-29
```

Comment:
This will need updating along with the file name when a publishing date is determined.